src/harness/virtualFileSystemWithWatch.ts

```diff
@@ -207,237 +211,56 @@ interface Array<T> { length: number; [n: number]: T; }`
         }
     }
 
-    interface CallbackData {

```

Comment:
This shouldnt be moved to vfs since its not needed in vfs (vfs doesnt have setTImeout and setImmediate timeout etc)

Reply:
It's not moved, I just couldn't convince the github diff algorithm not to display it as deleted+added.